### DISCUSSION TITLE
DIVISIONE QUADRIMESTRE

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#97] [Idee e proposte]
Buongiorno, sarebbe possibile dividere anche per quanto riguarda il lato genitori/alunni la sezione in due ,ossia una dove si riporta tutto ciò che riguarda il primo quadrimestre  ( assenze,ritardi,uscite,note,voti,argomenti, pagelle) e un'altra che riguarda solo il secondo quadrimestre. Giusto per non creare confusione

**Comment by trinko** [comment]
La metto tra le funzionalità richieste.
